styles.css

```diff
@@ -0,0 +1,113 @@
+* {
```

Comment:
Adding an html font size of 200% here could help with accessibility/developer empathy. Let's discuss considering implementation.

> WCAG Guideline 1.4.4—Except for captions and images of text, text can be resized without assistive technology up to 200 percent without loss of content or functionality.
When using CSS, most font sizes should be made in relative terms (e.g. %, em). This allows fonts to be displayed relative to a particular monitor setting or device.
source: https://accessibility.psu.edu/fontsizehtml/